tests/mock_module/api.py

```diff
@@ -64,10 +73,48 @@ class Record(RecordBase):
         delete=False,
     )
 
+    media_files = FilesField(
+        key="media_files",
+        bucket_id_attr="media_bucket_id",
```

Comment:
I fear this value is duplicated here and then in the `FilesOptionsComponent`, what would happen if there is a miss configuration and the values do not match? I have the gut feeling that they should be somehow centralized (i.e. rely on the systemfield or in the component, but not in both).